Assets/Reflex/Scripts/Configuration/ReflexConfiguration.cs

```diff
@@ -0,0 +1,17 @@
+using Reflex.Scripts.Logging;
+using UnityEngine;
+
+namespace Assets.Reflex.Scripts.Configuration
+{
+    public class ReflexConfiguration : ScriptableObject
+    {
+        public static LogLevel LogLevel { get; private set; }
```

Comment:
This static property is only set when you select the configuration and change the _logLevel serialized property.
So without doing it will force it to fallback. Can we make it non-static?

Reply:
Also, can ReflexConfiguration be non-required?
If you can find a single instance of it, use it, other wise, fallback to the default LogLevel

Reply:
The configuration SO is already non-required, because LogLevel property is static and it returns default value until changed. It would be nice to also make this property non-static but then we will need to pass SO instance to the logger class somehow.

I wanted SO instance to set LogLevel static property when playmode starts but OnValidate indeed doesn't work like that. What about https://docs.unity3d.com/2021.1/Documentation/ScriptReference/EditorApplication-playModeStateChanged.html instead?

Reply:
it may work, but lets keep it simple, lets remove all statics from the ReflexConfiguration and then, ensure when it gets created, it gets created under Assets/Resouces. (DOTween does the same way for instance).
And actually ProjectContext is loaded the same way, so we can follow the same pattern.

Reply:
Now loading config with `ResourcesUtilities.TryLoad`. Is that what you meant?

Reply:
Yes